cli/src/main/java/hudson/cli/SSHCLI.java

```diff
@@ -86,12 +90,9 @@ static int sshConnection(String jenkinsUrl, String user, List<String> args, Priv
 
         try (SshClient client = SshClient.setUpDefaultClient()) {
 
-            KnownHostsServerKeyVerifier verifier = new DefaultKnownHostsServerKeyVerifier(new ServerKeyVerifier() {
-                @Override
-                public boolean verifyServerKey(ClientSession clientSession, SocketAddress remoteAddress, PublicKey serverKey) {
-                    CLI.LOGGER.log(Level.WARNING, "Unknown host key for {0}", remoteAddress.toString());
-                    return !strictHostKey;
-                }
+            KnownHostsServerKeyVerifier verifier = new DefaultKnownHostsServerKeyVerifier((clientSession, remoteAddress, serverKey) -> {
+                CLI.LOGGER.log(Level.WARNING, "Unknown host key for {0}", remoteAddress.toString());
+                return !strictHostKey;
```

Comment:
unrelated change?

Reply:
Reverted to original code now. Tried to implement unecessary lambda for the function on automated suggestion.